src/components/flashcard/Flashcard.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import "./Flashcard.css";
+
+const Flashcard = ({ question, answer }) => {
+  const [isFlipped, setIsFlipped] = useState(false);
+  const [showHint, setShowHint] = useState(false);
+
+  let mockData = {
+    question: "What is our project group number?",
+    answer: "4",
+    hint: "The group number is a single-digit number",
+    tags: ["code the dream", "practicum"],
+  };
+
+  const handleFlip = () => {
+    setIsFlipped(!isFlipped);
+  };
+
+  const handleHint = () => {
+    setShowHint(!showHint);
+  };
+
+  return (
+    <div className="card">
```

Comment:
It's good practice to remove any commented code before you create a PR and any console logs as well. Just cleans up the code a bit. You can remove lines 25 and 46 but go ahead and keep the console logs that are triggered by the click handlers in the buttons on the back of the card - those are fine since they are providing temporary functionality.

Reply:
Agree 100%. Thanks! Will update the code promptly.